cmd/system-probe/api/server.go

```diff
@@ -53,6 +54,8 @@ func StartServer(cfg *sysconfigtypes.Config, telemetry telemetry.Component, wmet
 	mux.Handle("/debug/vars", http.DefaultServeMux)
 	mux.Handle("/telemetry", telemetry.Handler())
 
+	mux.HandleFunc("/debug/ebpf_btf_loader_info", ebpf.HandleBTFLoaderInfo)
```

Comment:
Should this exist for all OSes, or linux only?

Reply:
The handler has a stub for windows which indicates it's not supported. Could I remove on windows 

`fb.AddFileFromFunc(filepath.Join("system-probe", "ebpf_btf_loader.log"), getSystemProbeBTFLoaderInfo)`

from `archive.go` via a windows stub? I was trying not to put a 404 into flare_creation.log which could give the appearance of a bug in the flare logic

Reply:
I think it doesn't even hit this endpoint because of `archive_nolinux.go` right? You can wrap this with `if runtime.GOOS == "linux" {}`

Reply:
Oh, you are totally right. Yes, I will move it

Reply:
Switched to condition the route on `if runtime.GOOS == "linux" {}`